cypress/integration/models/developer/applicationinventory/application.ts

```diff
@@ -185,6 +190,7 @@ export class Application {
             business?: string;
             tags?: Array<string>;
             comment?: string;
+            repoType?: string;
         },
         cancel = false
     ): void {
```

Comment:
selectRepoType() method can be used in create method as well .

Reply:
Looks good Karishma , too make the test more robust . Let''s create two applications one with git and another with svn.
First test will apply filter for git and check app1 is present and app2 is not .
Second test will apply svn filter and check if app2 is present and app1 is not .